src/user.js

```diff
@@ -1,8 +1,8 @@
 class User {
-  constructor(id, name, pantry) {
-    this.id = id;
-    this.name = name;
-    this.pantry = pantry;
+  constructor(user) {
+    this.id = user.id;
+    this.name = user.name;
+    this.pantry = user.pantry;
```

Comment:
I really like how you simplified this so that we will just have to pass in the user to the constructor. It will make the JS so much cleaner as a whole. Nice thinking!